src/current/releases/molt.md

```diff
@@ -18,6 +18,18 @@ To download the latest MOLT Fetch/Verify binary:
 
 {% include molt/molt-install.md %}
 
+## December 13, 2024
+
+MOLT Fetch/Verify 1.2.1 is [available](#installation).
+
+- MOLT Fetch users now can use [`--assume-role`]({% link molt/molt-fetch.md %}#global-flags) to specify a service account for assume role authentication to cloud storage. `--assume-role` must be used with `--use-implicit-auth`, or it will be ignored. 
```

Comment:
Do you think it's worthwhile to show an example of using this flag in the docs above or do you think documenting it in "global flags" is enough here?

Reply:
There is actually an example in the flags table, so I think that should suffice!

Reply:
Ahh I see it. You're right. That should be clear enough for folks.